src/InputView.js

```diff
@@ -1,21 +1,62 @@
-/**
- * 사용자로부터 입력을 받는 역할을 한다.
- */
+const { Console } = require("@woowacourse/mission-utils");
+const { MESSAGE, GAME_RESOURCE } = require("./constants/Constant");
+const BridgeRandomNumberGenerator = require("./BridgeRandomNumberGenerator");
+const Validate = require("./Validate");
+const BridgeMaker = require("./BridgeMaker");
+const OutputView = require("./OutputView");
+
 const InputView = {
-  /**
-   * 다리의 길이를 입력받는다.
-   */
-  readBridgeSize() {},
+  readBridgeSize(bridgeGame) {
+    Console.readLine(MESSAGE.INPUT_BRIDGE_LENGTH, (inputBridgeSize) => {
+      Console.print("");
+      Validate.isNumber(inputBridgeSize);
+      Validate.checkLength(inputBridgeSize);
+      let gameTryCount = 0;
+      bridgeGame.getCount(gameTryCount);
+      const bridge = BridgeMaker.makeBridge(
+        inputBridgeSize,
+        BridgeRandomNumberGenerator.generate
+      );
+      let userMoveArray = [];
```

Comment:
`userMoveArray`는 다시 사용하지는 않으니까 const로 만들어도 될 것 같아요!
```suggestion
      const userMoveArray = [];
```

Reply:
아 그렇네요 수정하겠습니다!